.env

```diff
@@ -17,8 +17,12 @@ DATABASE_URL=mysql://root@127.0.0.1/sylius_%kernel.environment%
 ###> symfony/messenger ###
 # Choose one of the transports below
 # MESSENGER_TRANSPORT_DSN=amqp://guest:guest@localhost:5672/%2f/messages
-MESSENGER_TRANSPORT_DSN=doctrine://default
```

Comment:
We only removed it here. I think it's safe to remove it across the rest of the configuration files to let the new projects know that `MESSENGER_TRANSPORT_DSN` is no longer supported